### ISSUE TITLE
Checklists for Audio and Video@@page

### ISSUE THREAD

**Issue by nrhsinclair** [w3c/wai-media-guide#74]
I see the explanation under Checklists for Audio and Video that "If there are no “A”s, then it is not included in WCAG," but I find it confusing that item would be listed and linked to a resource, when not included in WCAG.

for example, rather than including link to Captions under both the Pre-Recorded Audio only (with no A) and Live Audio Only with (AA) only include Captions under the section where it is included in WCAG and include the A(s). It seems a bit overly complex and requires too much thought from the end users as to why is it listed here without an A, when there is another link close by with the A. (unless I am missing the purpose).
https://wai-media-guide.netlify.com/design-develop/media/planning/

**Comment by shawna-slh** [comment]
Hi Norah. Thanks for the input!

The issue is that we are including things that are needed/useful for accessibility, that go beyond the requirements of WCAG.

I rewrote it to:

> The checklists below... include:
> * What is required in the Web Content Accessibility Guidelines (WCAG) standard at Level A, AA, and AAA. _([WCAG](#WCAG-standards) is explained below.))_
> * What is needed to meet user needs, beyond WCAG. If there are no "A"s, then it is not required in WCAG.

Does that clarify it sufficiently?

**Comment by nrhsinclair** [comment]
Yes, that rewording clarifies. Thank you.
